com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -93,11 +100,12 @@ public UnityRLInitParameters Initialize(CommunicatorInitParameters initParameter
                 {
                     var pythonCommunicationVersion = initializationInput.RlInitializationInput.CommunicationVersion;
                     var pythonPackageVersion = initializationInput.RlInitializationInput.PackageVersion;
-                    if (pythonCommunicationVersion != initParameters.unityCommunicationVersion)
+                    var unityCommunicationVersion = initParameters.unityCommunicationVersion;
+                    if (!CheckCommunicationVersionsAreCompatible(unityCommunicationVersion, pythonCommunicationVersion))
```

Comment:
Should we also warn if they're compatible but different?

Reply:
yeah, I'll add a warning.